src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -9,9 +9,6 @@ open Signature_lib
 open Init
 module YJ = Yojson.Safe
 
-[%%check_ocaml_word_size
-64]
-
```

Comment:
should we remove this though? It seems like a good idea to have that check here as well, who knows what build system we'll use in the future

Reply:
This is a runtime check. If the code has been built for 64-bits, and the runtime platform is 32-bits, it won't run, I believe.

You're right, a different build system should check the OCaml config for word size.